Création classe Leclerc

MarcSuteau opened the pull request.

MarcSuteau committed:
modification du nom de détaillant

Tommy56 committed:
modification prix

Tommy56 committed:
creation d'un nouveau package + classe

MarcSuteau committed:
création d'un setter pour la commande + création fonction commande à
appeler avant passage au step suivant

Tommy56 committed:
setPrix

MarcSuteau committed:
Version préliminaire de l'acteur Leclerc

MarcSuteau committed:
version préliminaire de acteur Leclerc

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/ActeurLeclerc.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Tommy56 committed:
Ajout de commentaires

MarcSuteau committed:
classe Leclerc

MarcSuteau committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Tommy56 committed:
peupler mondeV1

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/MondeV1.java

massimotisi commented:
Je viens d'accepter la pull request du groupe 2 qui est en conflict avec votre pull request. Pouvez-vous re-synchroniser avec mon repository? 1) fetch from... 2) merge 3) add to index 4) commit 5) push

MarcSuteau commented:
Je viens de le faire mais il n'y a aucun changement donc impossible de push

massimotisi commented:
Les etapes detaillées:
1) "team/remote/fetch from..." verifier que c'est mon address https://github.com/CACAO2016/CACAO2016.git
2) "team/merge" et choisir le branch que vous venez de telecharger (si vous n'avez pas specifié de nom pour le branch c'est "choose a name...")
3) si il y a des conflits modifier à la main les fichiers et add to index
4) commit 
5) push

MarcSuteau commented:
C'est bien ce que j'ai essayé de faire mais ils n'y a rien que je puisse fetch

massimotisi commented:
Dans la fenetre du fetch il faut cliquer sur les deux boutons "Add All Branches Spec" et "Add All Tags Spec", et aprés "Finish". Si le fetch est correct vous allez voire une petite fleche à coté du nom de votre projet avec le nombre de commit entrants.

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/MondeV1.java

MarcSuteau commented:
Ah oui voilà j'ai réussi à push ca devrait être bon normalement

massimotisi commented:
C'est ok, mais vous avez committé une nouvelle version des classes "PRIME_CACAO_2016/src/abstraction/fourni/v0/Constantes.java" et "PRIME_CACAO_2016/src/abstraction/fourni/v0/MondeV0.java"

Vous n'avez pas le droit de changer les classes dans le dossier " PRIME_CACAO_2016/src/abstraction/fourni/v0".

Je vous conseil de le remplacer avec la version qui se trouve dans mon repository, aprés committer et pusher encore.

MarcSuteau committed:
rectification du nombre de step par an pour se caler avec les autres

MarcSuteau committed:
rectification de constantes v0

MarcSuteau commented:
Voilà j'ai remis le nom initial

massimotisi commented:
Vous avez bien corrigé le fichier "PRIME_CACAO_2016/src/abstraction/fourni/v0/Constantes.java" mais pas "PRIME_CACAO_2016/src/abstraction/fourni/v0/MondeV0.java".

MarcSuteau committed:
rectification de monde v0

massimotisi commented:
Je viens d'accepter la pull request du groupe 4 qui est en conflict avec votre pull request. Pouvez-vous re-synchroniser avec mon repository? 1) fetch from... 2) merge 3) add to index 4) commit 5) push

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/MondeV1.java

MarcSuteau commented:
Voilà

massimotisi commented:
Merci, mais je ne peux pas encore accepter parce-que quand j'execute votre fork j'ai l'exception suivante et la fenetre ne s'affiche pas:

Exception in thread "main" java.lang.NullPointerException
    at abstraction.fourni.Monde.getActeur(Monde.java:102)
    at abstraction.fourni.Monde.ajouterActeur(Monde.java:68)
    at abstraction.commun.MondeV1.peupler(MondeV1.java:34)
    at presentation.FenetrePrincipale.<init>(FenetrePrincipale.java:43)
    at presentation.FenetrePrincipale.main(FenetrePrincipale.java:146)

MarcSuteau commented:
Je dois faire une autre manipulation du coup ?

massimotisi commented:
Bien sur! 

Si vous executez le programme (la methode main de la classe presentation/FenetrePrincipale.java) dans le code de mon repository (qui contient le code des autres groupes) vous aller voir la fenetre et l'evolution des indicateurs. Par contre si vous executez la meme methode dans votre code on vois juste une exception. Je n'accepte pas du code qui leve d'exceptions et surtout pas du code qui casse tout le programme!!

MarcSuteau committed:
rajout du this.nom au constructeur Leclerc qui faisait planter le
programme, en attente d'une correction pour que les indicateurs
fonctionnent correctement

MarcSuteau committed:
correction de l'indicateur solde, on retirait des sous aux
transformateurs au lieu de leur en donner, et notre propre solde ne
bougeait pas

MarcSuteau commented:
Le programme ne plante plus et les indicateurs semblent fonctionner normalement maintenant

massimotisi commented:
Ok, j'accepte

florianed commented:
Bonjour, 
je n'arrive pas à faire ma pull request final. Je ne sais pas pourquoi, voici l'écran que j'obtiens.
Merci d'avance, 
Floriane
Date: Wed, 27 Apr 2016 11:32:22 -0700
From: notifications@github.com
To: CACAO2016@noreply.github.com
CC: 
Subject: Re: [CACAO2016/CACAO2016] Création classe Leclerc (#8)

Merged #8.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
